### PR TITLE
[4.0] Remove obsolete menu class "nav-pills" from menu modules' parameters and contact links

### PR DESCRIPTION
**Pull request by richard67**
Pull Request for Issue #23023 .

### Summary of Changes

Remove the obsolete menu class "nav-pills" from contact default links markup and blog sampledata plugin menu creation.

### Testing Instructions

Code review for the sample data part, patch tester and check that contact links look like without this PR for the contact links part.

### Expected result

The menu class "nav-pills" is not added in blog sample data for menus at module position "top-a", e.g. the "Main Menu Blog", and also not added to the markup of the contact links list (`<ul>` element).

### Actual result

The menu class "nav-pills" is added in blog sample data for menus at module position "top-a", e.g. the "Main Menu Blog", but it has no effect.
This might confuse administrators who create or maintain menu modules.
The CSS class nav-pills" is also added to the markup of the contact links list (`<ul>` element), but it has no effect there because the links list doesn't have an active element on which the pills effect should be applied.

### Documentation Changes Required

None.

### CODE REVIEW

**Comment by richard67** [comment]
@ciar4n Could you review or test this PR?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23055">issues.joomla.org/tracker/joomla-cms/23055</a>.</sub>

File: components/com_contact/tmpl/contact/default_links.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
 <?php endif; ?>
 
 <div class="com-contact__links contact-links">
-	<ul class="nav nav-pills flex-column">
```
Comment (ciar4n) [inline components/com_contact/tmpl/contact/default_links.php:28]: The reason for `nav-pills` not working from the database should not apply here.

Comment (richard67) [reply components/com_contact/tmpl/contact/default_links.php:28]: Well, but this class has no effect in the Cassiopeia template. Compare the look of contact links with and without nav-pills, and you will not see any difference. That's why I think it should be removed here, too.

Comment (ciar4n) [reply components/com_contact/tmpl/contact/default_links.php:28]: I see now. Bootstraps `nav-pills` class applies the pill effect to 'active' items only. There is no 'active' items in this list so you are correct.

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on 152e18326c626012579edcccc7db6aa181e64db0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23055">issues.joomla.org/tracker/joomla-cms/23055</a>.</sub>

**Comment by richard67** [comment]
@ciar4n Could you set your test result again? It has been reset by branch update to latest 4.0-dev. There have not been made any changes. Thanks in advance.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23055">issues.joomla.org/tracker/joomla-cms/23055</a>.</sub>

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on 9c2f0721b39905754bafe99937147d6b8b92c268<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23055">issues.joomla.org/tracker/joomla-cms/23055</a>.</sub>

**Comment by richard67** [comment]
Thanks a lot. Hope we find a 2nd tester.

**Comment by richard67** [comment]
@ciar4n Sorry, could you set your test result again? I had again to update branch. Thanks in advance.
@Quy Do you have time and mood to test this PR?

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 5dcd2da4235c731c7878d68d9b7352db0897f6ef<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23055">issues.joomla.org/tracker/joomla-cms/23055</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23055">issues.joomla.org/tracker/joomla-cms/23055</a>.</sub>

**Comment by infograf768** [comment]
Updated @ciar4n test result in Issues.
Relaunched Travis to try to make it pass as the analysis4x issue is not related.

**Comment by richard67** [comment]
@infograf768 Thanks a lot.

**Comment by infograf768** [comment]
@richard67 
Maybe you have time to test:
https://github.com/joomla/joomla-cms/pull/23143 (RTL stuff too)

**Comment by wilsonge** [comment]
Thanks guys! Sorry for the delay on merging

**Pull request merged**